src/routes/search.py

```diff
@@ -25,17 +25,25 @@ def get_blueprint():
     return search_blueprint
 
 
-@search_blueprint.route("/search_ingredients")
+
+@search_blueprint.route("/search_ingredients", methods=["POST"])
 def search_ingredients():
-    return render_template("search.html")
+    if flask.request.method == "POST":
+        ingredients = flask.request.form.get("searchIngredients")
+        # prints what the user searches for to the terminal
+        print(ingredients)
+        returnedDict = api.spoonacular.search_ingredients(ingredients)
```

Comment:
The top comment "import spoonacular somehow" is too generic and needs to be updated with "from api.spoonacular import search_ingredients"